### PR TITLE
[OP#41487] Replace icons for no connection and add link

### PR DESCRIPTION
**Pull request by SwikritiT**
OP#41487
https://community.openproject.org/projects/nextcloud-integration/work_packages/41487/activity

Figma links:
https://www.figma.com/file/gtLQfPe09X7XugAH8L7dTy/Nextcloud-Integration?node-id=31%3A6555
https://www.figma.com/file/gtLQfPe09X7XugAH8L7dTy/Nextcloud-Integration?node-id=746%3A31002

The current UI looks like this

## Light Mode

noconLight
addLinkLight
conOPLight

## Dark Mode
noConDark
addLinkDark
conOPDark

### CODE REVIEW

File: tests/jest/components/tab/EmptyContent.spec.js [hunk 1]
```diff
@@ -42,14 +42,14 @@ describe('EmptyContent.vue Test', () => {
...
-			message: 'No workspaces linked yet',
+			message: 'No OpenProject links yet To add a link, use the search bar above to find the desired work package',
 		}, {
 			state: 'something else',
 			message: 'invalid state',
 		}])('shows the correct empty message depending on states if the request url is valid', async (cases) => {
 			wrapper = getWrapper({ state: cases.state, adminConfigStatus: true })
 			expect(wrapper.find(emptyContentMessageSelector).exists()).toBeTruthy()
-			expect(wrapper.find(emptyContentMessageSelector).text()).toMatch(cases.message)
+			expect(wrapper.find(emptyContentMessageSelector).text().replace(/[\t\n\r]/gm, '')).toMatch(cases.message)
```
Comment (SwikritiT) [inline tests/jest/components/tab/EmptyContent.spec.js:52]: due to the title and subtitle differentiation the message comes with tabs and new lines so regex is added to replace the tabs and new lines
```
"No OpenProject links yet·····
    				To add a link, use the search bar above to find the desired work package"
```

File: src/components/OAuthConnectButton.vue [hunk 2]
```diff
@@ -83,9 +84,11 @@ export default {
 <style lang="scss" scoped>
 .oauth-connect {
 	&--message {
-		font-size: .875rem;
+		font-size: 1rem;
 		text-align: center;
-		font-weight: 500;
+		font-weight: 400;
+		color: #878787;
+		padding: 0px 18px 0px 18px;
```
Comment (kiranparajuli589) [inline src/components/OAuthConnectButton.vue:91]:
```suggestion
		padding: 0px 18px;
```

File: src/components/tab/EmptyContent.vue [hunk 3]
```diff
@@ -82,28 +97,27 @@ export default {
...
+			font-weight: 600;
+			padding: 4px 12px 12px 12px;
+			color: #333333;
+		}
+		&--sub-title{
+			font-size: 1rem;
+			text-align: center;
+			font-weight: 400;
+			color: #878787;
+			padding: 0px 18px 0px 18px;
```
Comment (kiranparajuli589) [inline src/components/tab/EmptyContent.vue:119]:
```suggestion
			padding: 0px 18px;
```

File: src/components/tab/EmptyContent.vue [hunk 4]
```diff
@@ -44,26 +50,35 @@ export default {
 		}
 	},
 	computed: {
-		flowSvg() {
-			return require('../../../img/flow.svg')
```
Comment (kiranparajuli589) [inline src/components/tab/EmptyContent.vue:48]: the actual file can also be removed. IMO it's not being used anywhere else

File: tests/jest/components/tab/EmptyContent.spec.js (hunk 1 above)
Comment (kiranparajuli589) [reply tests/jest/components/tab/EmptyContent.spec.js:52]: you can use the snapshot to avoid such case, also would be more reliable for the update

Comment (SwikritiT) [reply tests/jest/components/tab/EmptyContent.spec.js]: done

File: tests/jest/components/tab/__snapshots__/EmptyContent.spec.js.snap [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  <div class="empty-content--message--sub-title">
+    To add a link, use the search bar above to find the desired work package
+  </div>
+</div>
+`;
+
+exports[`EmptyContent.vue Test content title shows the correct empty message depending on states if the request url is valid 6`] = `
+<div class="empty-content--message">
+  <div class="empty-content--message--title">
+    invalid state
```
Comment (kiranparajuli589) [inline tests/jest/components/tab/__snapshots__/EmptyContent.spec.js.snap:53]: is this the correct message @SwikritiT?

Comment (SwikritiT) [reply tests/jest/components/tab/__snapshots__/EmptyContent.spec.js.snap:53]: yes, it is. If the state is something other than the specified. If you have any better suggestion I can change the message

https://github.com/nextcloud/integration_openproject/blob/54821b5867bd2d776fe9795f866f65609b8774b4/src/components/tab/EmptyContent.vue#L53-L65

Comment (kiranparajuli589) [reply tests/jest/components/tab/__snapshots__/EmptyContent.spec.js.snap:53]: oh yes, it's for the last else clause. :+1:

Comment (SwikritiT) [reply tests/jest/components/tab/__snapshots__/EmptyContent.spec.js.snap]: Sounds nice. Did it the latest commit

**Review by kiranparajuli589 — COMMENTED** [review]
LGTM :+1:

**Review by individual-it — APPROVED** [review]

**Pull request merged**